main/ble2mqtt.c

```diff
@@ -437,11 +443,11 @@ static void _ble_on_mqtt_set(const char *topic, const uint8_t *payload,
     size_t len, void *ctx);
 
 static void ble_on_characteristic_found(mac_addr_t mac, ble_uuid_t service_uuid,
-    ble_uuid_t characteristic_uuid, uint8_t properties)
+    ble_uuid_t characteristic_uuid, uint16_t index, uint8_t properties)
 {
-    ESP_LOGD(TAG, "Found new characteristic: service: " UUID_FMT
-      ", characteristic: " UUID_FMT ", properties: 0x%x",
-      UUID_PARAM(service_uuid), UUID_PARAM(characteristic_uuid), properties);
+    ESP_LOGI(TAG, "Found new characteristic: service: " UUID_FMT
+      ", characteristic: " UUID_FMT " index %d, properties: 0x%x",
```

Comment:
Index is an unsigned, please use the correct format specifier (`PRIu16`, in this case) and add a comma after the characteristic UUID